mdtools/box.py

```diff
@@ -695,46 +689,46 @@ def unwrap_trj(topfile, trjfile, universe, atm_grp, end=-1,
             compound != 'segments' and
             compound != 'residues' and
             compound != 'molecules' and
-            compound != 'fragments'):
+                compound != 'fragments'):
             raise ValueError("compound must be either 'group',"
                              " 'segments', 'residues', 'molecules' or"
                              " 'fragments', but you gave '{}'"
                              .format(compound))
         if (make_whole and
             reference != 'com' and
             reference != 'cog' and
-            reference is not None):
+                reference is not None):
             raise ValueError("reference must be either 'com', 'cog' or"
                              " None, but you gave {}".format(reference))
         if make_whole and reference == 'com':
             mdt.check.masses(ag=atm_grp, flash_test=False)
         if make_whole and len(atm_grp.bonds) <= 0:
             raise ValueError("The AtomGroup contains no bonds")
-    
+
     if end < 0:
         end = universe.trajectory.n_frames
     _, end, _, _ = mdt.check.frame_slicing(
-                       start=0,
-                       stop=end,
-                       step=1,
-                       n_frames_tot=universe.trajectory.n_frames)
+        start=0,
+        stop=end,
+        step=1,
+        n_frames_tot=universe.trajectory.n_frames)
     displacement = np.zeros(atm_grp.positions.shape, dtype=np.float64)
-    
+
     ts = universe.trajectory[0]
     if verbose:
         timer = datetime.now()
         proc = psutil.Process(os.getpid())
-        last_frame = universe.trajectory[end-1].frame
+        last_frame = universe.trajectory[end - 1].frame
```

Comment:
Yeah, this is totally debatable and I also prefer the denser notation. But this is also the way that black formats things. My best guess is that this is because of [this recommendation.](https://www.python.org/dev/peps/pep-0008/#other-recommendations) but this does not mention indices.

Reply:
Yes, there should be extra specifications for indices. But hey, that's how it is and I think the benefits of a code formatter are higher than small personal preferences.